docs/api.md

```diff
@@ -4,6 +4,7 @@
   * [options](#options)
   * [destination](#destination)
   * [destination\[Symbol.for('pino.metadata')\]](#metadata)
+  * [destination: pino-multi-stream](#multi-stream)
```

Comment:
I do not think this belongs in the API documentation because it doesn't really have anything to do with the core API. If anything, this is an example showing how to use an ecosystem module to provide a stream.

Reply:
I have checked for other section but cannot find a better place. I agree with you that that's not the best place for it but I also think that most users coming from other loggers may find that really useful!

Reply:
I think this might be the best spot: https://github.com/pinojs/pino/blob/master/docs/help.md

Reply:
I'm missing somethin, I thought that the first element of pino-multi-stream was for the `default` stream but that is only for the `info` level. Is there a way to set a standard stream for levels that doesn't have a defined stream?

What I would like to do is simply make all streams go to stdout and fatal and error to stderr